autorest/codegen/templates/templates_package_dataplane/swagger/README.md.jinja2

```diff
@@ -0,0 +1,20 @@
+### Settings
+
+```yaml
+input-file:
+{% for file in code_model.options['input_file'] -%}
+- {{ file }}
+{% endfor -%}
+output-folder: {{ '../' + code_model.options["package_name"].replace('-', '/') }}
```

Comment:
I don't think we need to generate a swagger readme file as part of packaging... aren't they running the package mode from a config file already?